### PR TITLE
IMPROVE the build-storybooks script

### PR DESCRIPTION
**Pull request by ndelangen**

### CODE REVIEW

**Comment by ndelangen** [comment]
@gabrielcsapo @aromanarguello Would either of you be able to understand why the storybook for ember isn't building?

File: scripts/build-storybooks.js [hunk 1]
```diff
@@ -119,9 +119,14 @@ const createContent = deployables => {
...
   const deployables = files.filter(f => {
     const packageJsonLocation = p(['examples', f, 'package.json']);
-    const stats = statSync(packageJsonLocation);
-
-    return stats.isFile() && hasBuildScript(packageJsonLocation);
+    let stats = null;
+    try {
+      stats = statSync(packageJsonLocation);
+    } catch (e) {
+      //
```
Comment (shilman) [inline scripts/build-storybooks.js]: log the error, or at least a summary

**Review by shilman — CHANGES_REQUESTED** [review]
minor changes

**Comment by ndelangen** [comment]
The now.sh failure is exactly the point of this PR, it IS already failing on next, but we're not seeing it because the error isn't caught in NodeJS 10

**Comment by ndelangen** [comment]
I think I'll temporarily disable the ember example to be build to see if this is the last broken example.

File: scripts/build-storybooks.js (hunk 1 above)
Comment (ndelangen) [reply scripts/build-storybooks.js]: The error it's hiding is when there are temp system files in the examples directory, which are not real examples.

I do not think it's productive to see those errors tbh.

WDYT?

Comment (shilman) [reply scripts/build-storybooks.js]: can you just print out: `Skipping: ${e.message}` or something?

**Review by shilman — APPROVED** [review]
LGTM

**Pull request merged**